receiver/cloudfoundryreceiver/converter.go

```diff
@@ -17,6 +19,31 @@ const (
 	attributeNamePrefix = "org.cloudfoundry."
 )
 
+var ResourceAttributesKeys = []string{
+	"index",
+	"ip",
+	"deployment",
+	"id",
+	"job",
+	"product",
+	"instance_group",
+	"instance_id",
+	"origin",
+	"system_domain",
+	"source_id",
+	"source_type",
+	"process_type",
+	"process_id",
+	"process_instance_id",
+}
+
+var allowResourceAttributes = featuregate.GlobalRegistry().MustRegister(
+	"cloudfoundry.resourceAttributes.allow",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("When enabled, envelope tags are copied to the metrics as resource attributes instead of datapoint attributes"),
+	featuregate.WithRegisterFromVersion("v0.109.0"),
```

Comment:
@odubajDT, sorry for the delay. While @crobert-1 is away, I'll help you with merging the PR. Can you please update this info and rebase the PR? Maybe we can go straight to beta?

Reply:
Hi sure, will update it. To be honest I would rather stick to alpha first (at least for standard 2 versios), so we won't break anything for anybody during the holiday season :)